sp800-63c/references.md

```diff
@@ -5,30 +5,43 @@
 
 *This section is informative.*
 
+### 12.1 General References
+<a name="FEDRAMP"></a>[FEDRAMP] General Services Administration, *Federal Risk and Authorization Management Program*, available at: <https://www.fedramp.gov/>.
+
 <a name="M-03-22"></a>[M-03-22] OMB Memorandum M-03-22, *OMB Guidance for Implementing the Privacy Provisions of the E-Government Act of 2002*, September 26, 2003, available at: <https://georgewbush-whitehouse.archives.gov/omb/memoranda/m03-22.html>.
 
-<a name="RFC7591"></a>[RFC 7591] IETF, *OAuth 2.0 Dynamic Client Registration Protocol*, available at: <https://tools.ietf.org/html/rfc7591/>.
+<a name="nistir8112"></a>[NISTIR 8112] NIST Internal Report 8112 (Draft), *Attribute Metadata*, available at: <https://pages.nist.gov/NISTIR-8112/NISTIR-8112.html>.
 
-<a name="RFC7636"></a>[RFC 7636] IETF, *Proof Key For Code Exchange*, available at: <https://tools.ietf.org/html/rfc7636/>.
+<a name="Section508"></a>[Section 508] Section 508 Law and Related Laws and Policies (January 30, 2017), available at: <https://www.section508.gov/content/learn/laws-and-policies>.
 
-<a name="nistir8112"></a>[NISTIR 8112] NIST Internal Report 8112, *Attribute Metadata*, available at: <https://pages.nist.gov/NISTIR-8112/NISTIR-8112.html>.
+### 12.2 Standards
+<a name="bcp195"></a>[BCP 195] Sheffer, Y., Holz, R., and P. Saint-Andre, *Recommendations for Secure Use of Transport Layer Security (TLS) and Datagram Transport Layer Security (DTLS)*, BCP 195, RFC 7525,DOI 10.17487/RFC7525, May 2015, <https://doi.org/10.17487/RFC7525>.
 
-<a name="ISO9241-11"></a>[ISO 9241-11] International Organization for Standardization, *Ergonomic requirements for office work with visual display terminals (VDTs) — Part 11: Guidance on Usability*, ISO 9241-11:1998, ISO: Geneva, Switzerland, 1998.
+<a name="ISO9241"></a>[ISO 9241-11] International Standards Organization, ISO/IEC 9241-11 *Ergonomic requirements for office work with visual display terminals (VDTs) — Part 11: Guidance on usability*, 1998, available at: <https://www.iso.org/standard/16883.html>.
 
-<a name="Section508"></a>[Section 508] Section 508 Law and Related Laws and Policies (January 30, 2017), available at: <https://www.section508.gov/content/learn/laws-and-policies>.
+<a name="OpenIDConnectCore"></a>[OpenID Connect Core 1.0] Sakimura, N., Bradley, B., Jones, M., de Medeiros, B., and C. Mortimore, *OpenID Connect Core 1.0 incorporating errata set 1*, November, 2014. Available at: <https://openid.net/specs/openid-connect-core-1_0.html>.
```

Comment:
Previous anchor was "OIDC" and new anchor is "OpenIDConnectCore" but the references to that anchor weren't updated to use the new anchor.